tools/nuget/generate_nuspec_for_native_nuget.py

```diff
@@ -213,6 +227,11 @@ def generate_files(list, args):
                           '" target="lib\\netstandard2.0\\Microsoft.AI.MachineLearning.Interop.pdb" />')
 
     # Process runtimes
+    # Process linux
+    if (args.linux_build != 'None'):
+        files_list.append('<file src=' + '"' + os.path.join(args.sources_path, args.linux_build) +
```

Comment:
args.linux_build is now a bool right? 
so it should be os.path.join(args.sources_path, 'libonnxruntime.so') instead?